src/AzureAuth/Commands/CommandAad.cs

```diff
@@ -85,7 +86,7 @@ public class CommandAad
         /// <summary>
         /// The default number of minutes CLI is allowed to run.
         /// </summary>
-        public static readonly TimeSpan GlobalTimeout = TimeSpan.FromMinutes(15);
+        public static readonly TimeSpan GlobalTimeout = 15.0.Minutes();

```

Comment:
[nit] without extention.
```suggestion
        public static readonly TimeSpan GlobalTimeout = TimeSpan.FromMinutes(15.0);
```